src/commands/edge_app.rs

```diff
@@ -453,13 +453,24 @@ impl EdgeAppCommand {
             let value = commands::get(
                 &self.authentication,
                 &format!(
-                    "v4/assets?select=status&app_id=eq.{}&app_revision=eq.{}&status=neq.finished",
+                    "v4/assets?select=status,processing_error,title&app_id=eq.{}&app_revision=eq.{}&status=neq.finished",
                     app_id, revision
                 ),
             )?;
             debug!("ensure_assets_processing_finished: {:?}", &value);
 
             if let Some(array) = value.as_array() {
+                for item in array {
+                    if let Some(status) = item["status"].as_str() {
+                        if status == "error" {
+                            return Err(CommandError::AssetProcessingError(format!(
```

Comment:
Let's maybe switch to anyhow for new code ? We already have a gazillion of error types and it's not helpful anywhere to be honest